EntityFishingZombie attack crash

#### Which modpack version are you using?
2.0.8.4
#
#### If in multiplayer; On which server does this happen? 

#
#### What did you try to do, and what did you expect to happen?
walking
#
#### What happened instead? (Attach screenshots if needed)
crash
#
#### What do you suggest instead/what changes do you propose?
https://github.com/GTNewHorizons/SpecialMobs/commit/59a5afefdb243b063eedf349503429eb3db3dfee#diff-fdb8c8d0cb0c937df0459bee8a991e9f
# 
### Resume
Basically when a FishingZombie makes his special attack, minecraft crash
#
### this is the crash report:

**Description: Ticking entity**

java.lang.ExceptionInInitializerError
       at toast.specialMobs.entity.zombie.EntityFishingZombie.func_70636_d(EntityFishingZombie.java:96)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1611)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:272)
	at net.minecraft.entity.monster.EntityMob.func_70071_h_(SourceFile:30)
	at net.minecraft.entity.monster.EntityZombie.func_70071_h_(EntityZombie.java:303)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:648)
	at net.minecraft.world.World.func_72870_g(World.java:2034)
	at WorldServerOF.func_72870_g(WorldServerOF.java:347)
	at net.minecraft.world.World.func_72939_s(World.java:1887)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:186)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:396)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
Caused by: java.lang.NullPointerException
	at toast.specialMobs.Properties.getBoolean(Properties.java:302)
	at toast.specialMobs.Properties.getBoolean(Properties.java:291)
	at toast.specialMobs.entity.EntitySpecialFishHook.<clinit>(EntitySpecialFishHook.java:21)
	... 16 more

**A detailed walkthrough of the error, its code path and all known details is as follows:**

-- Head --
Stacktrace:
	at toast.specialMobs.entity.zombie.EntityFishingZombie.func_70636_d(EntityFishingZombie.java:96)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1611)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:272)
	at net.minecraft.entity.monster.EntityMob.func_70071_h_(SourceFile:30)
	at net.minecraft.entity.monster.EntityZombie.func_70071_h_(EntityZombie.java:303)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:648)
	at net.minecraft.world.World.func_72870_g(World.java:2034)
	at WorldServerOF.func_72870_g(WorldServerOF.java:347)
	at net.minecraft.world.World.func_72939_s(World.java:1887)

-- Entity being ticked --
Details:
	Entity Type: SpecialMobs.FishingZombie (toast.specialMobs.entity.zombie.EntityFishingZombie)
	Entity ID: 57763
	Entity Name: Fishing Zombie
	Entity's Exact location: -549.74, 69.00, 410.41
	Entity's Block location: World: (-550,69,410), Chunk: (at 10,4,10 in -35,25; contains blocks -560,0,400 to -545,255,415), Region: (-2,0; contains chunks -64,0 to -33,31, blocks -1024,0,0 to -513,255,511)
	Entity's Momentum: 0.05, -0.08, 0.00
Stacktrace:
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636)

-- Affected level --
Details:
	Level name: New World
	All players: 1 total; [EntityPlayerMP['Seitenca'/245, l='New World', x=-541.57, y=71.00, z=406.02]]
	Chunk stats: ServerChunkCache: 627 Drop: 0
	Level seed: -376439466835431966
	Level generator: ID 04 - RWG, ver 0. Features enabled: true
	Level generator options: 2;1x7,52x24,1x3:2,;2;biome_1,village
	Level spawn location: World: (-83,64,176), Chunk: (at 13,4,0 in -6,11; contains blocks -96,0,176 to -81,255,191), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Level time: 12967 game time, 45436 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 98252 (now: false), thunder time: 81655 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: true
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:186)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:396)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

@Dream-Master I have the same problem when I hid a Fishing Zombie

`---- Minecraft Crash Report ----
// Shall we play a game?

Time: 2/15/20 8:51 PM
Description: Ticking entity

java.lang.ExceptionInInitializerError
	at toast.specialMobs.entity.zombie.EntityFishingZombie.func_70636_d(EntityFishingZombie.java:96)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1611)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:206)
	at net.minecraft.entity.monster.EntityMob.func_70071_h_(SourceFile:30)
	at net.minecraft.entity.monster.EntityZombie.func_70071_h_(EntityZombie.java:303)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:648)
	at net.minecraft.world.World.func_72870_g(World.java:2034)
	at net.minecraft.world.World.func_72939_s(World.java:1887)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
Caused by: java.lang.NullPointerException
	at toast.specialMobs.Properties.getBoolean(Properties.java:302)
	at toast.specialMobs.Properties.getBoolean(Properties.java:291)
	at toast.specialMobs.entity.EntitySpecialFishHook.<clinit>(EntitySpecialFishHook.java:21)
	... 15 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at toast.specialMobs.entity.zombie.EntityFishingZombie.func_70636_d(EntityFishingZombie.java:96)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1611)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:206)
	at net.minecraft.entity.monster.EntityMob.func_70071_h_(SourceFile:30)
	at net.minecraft.entity.monster.EntityZombie.func_70071_h_(EntityZombie.java:303)
	at net.minecraft.world.World.func_72866_a(World.java:2070)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:648)
	at net.minecraft.world.World.func_72870_g(World.java:2034)

-- Entity being ticked --
Details:
	Entity Type: SpecialMobs.FishingZombie (toast.specialMobs.entity.zombie.EntityFishingZombie)
	Entity ID: 269
	Entity Name: Fishing Zombie
	Entity's Exact location: 437.76, 50.00, -1927.15
	Entity's Block location: World: (437,50,-1928), Chunk: (at 5,3,8 in 27,-121; contains blocks 432,0,-1936 to 447,255,-1921), Region: (0,-4; contains chunks 0,-128 to 31,-97, blocks 0,0,-2048 to 511,255,-1537)
	Entity's Momentum: 0.02, -0.08, -0.03
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1887)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)

-- Affected level --
Details:
	Level name: world
	All players: 2 total; [EntityPlayerMP['WhatsTheBadNews'/264, l='world', x=439.82, y=50.00, z=-1923.70], EntityPlayerMP['Apollsyne'/909, l='world', x=424.52, y=50.00, z=-1920.87]]
	Chunk stats: ServerChunkCache: 774 Drop: 0
	Level seed: -8396616355261351257
	Level generator: ID 04 - RWG, ver 0. Features enabled: true
	Level generator options: 
	Level spawn location: World: (58,64,-84), Chunk: (at 10,4,12 in 3,-6; contains blocks 48,0,-96 to 63,255,-81), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,0,-512 to 511,255,-1)
	Level time: 53003183 game time, 57403624 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 10822 (now: true), thunder time: 115110 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Linux (amd64) version 4.15.0-54-generic
	Java Version: 1.8.0_242, Private Build
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Private Build
	Memory: 6955573472 bytes (6633 MB) / 10488905728 bytes (10003 MB) up to 10488905728 bytes (10003 MB)
	JVM Flags: 2 total; -Xmx10240M -Xms10240M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 225 mods loaded, 225 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
	UCHIJAAAA	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
	UCHIJAAAA	appliedenergistics2-core{rv3-beta-22} [Applied Energistics 2 Core] (minecraft.jar) 
	UCHIJAAAA	CodeChickenCore{1.0.7.47} [CodeChicken Core] (minecraft.jar) 
	UCHIJAAAA	CropLoadCoreASM{0.0.1} [CroploadCore ASM Core] (minecraft.jar) 
	UCHIJAAAA	Micdoodlecore{} [Micdoodle8 Core] (minecraft.jar) 
	UCHIJAAAA	NotEnoughItems{2.0.0-beta-6-GTNH} [Not Enough Items] (NotEnoughItems-1.7.10-2.0.0-beta-6-GTNH-universal.jar) 
	UCHIJAAAA	OpenComputers|Core{1.7.5.1+4f95ff4} [OpenComputers (Core)] (minecraft.jar) 
	UCHIJAAAA	PlayerAPI{1.4} [Player API] (minecraft.jar) 
	UCHIJAAAA	ThE-core{1.0.0.1} [Thaumic Energistics Core] (minecraft.jar) 
	UCHIJAAAA	ThaumicTinkerer-preloader{0.1} [Thaumic Tinkerer Core] (minecraft.jar) 
	UCHIJAAAA	WitchingGadgetsCore{1.2.10-GTNH} [Witching Gadgets Core] (minecraft.jar) 
	UCHIJAAAA	OpenModsCore{0.10} [OpenModsCore] (minecraft.jar) 
	UCHIJAAAA	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) 
	UCHIJAAAA	GT++_Preloader{0.4-Beta} [GT++ Preloader] (minecraft.jar) 
	UCHIJAAAA	BWCore{0.0.1} [BartWorks ASM Core] (minecraft.jar) 
	UCHIJAAAA	battlegear2{1.7.10} [Mine & Blade Battlegear 2 - Bullseye] (1.7.10-MB_Battlegear2-Bullseye-1.0.8.5.jar) 
	UCHIJAAAA	IC2{2.2.828-experimental} [IndustrialCraft 2] (industrialcraft-2-2.2.828a-experimental.jar) 
	UCHIJAAAA	AdvancedSolarPanel{1.7.10-3.5.1} [Advanced Solar Panels] (AdvancedSolarPanel-1.7.10-3.5.1a.jar) 
	UCHIJAAAA	adventurebackpack{1.7.10-0.9-h10} [Adventure Backpack] (adventurebackpack-1.7.10-0.9-h10.jar) 
	UCHIJAAAA	appliedenergistics2{rv3-beta-22} [Applied Energistics 2] (appliedenergistics2-rv3-beta-22.jar) 
	UCHIJAAAA	bdlib{1.9.4.109} [BD Lib] (bdlib-1.9.4.109-mc1.7.10.jar) 
	UCHIJAAAA	ae2stuff{0.5.1.9} [AE2 Stuff] (ae2stuff-rv3-0.5.1.9-mc1.7.10.jar) 
	UCHIJAAAA	AFSU{1.2.3a-Freeza} [AFSU Mod] (AFSU-1.2.3a-Freeza.jar) 
	UCHIJAAAA	YAMCore{0.5.78} [YAMCore] (YAMCore-1.7.10-0.5.78.jar) 
	UCHIJAAAA	Baubles{1.0.1.10} [Baubles] (Baubles-1.7.10-1.0.1.11.jar) 
	UCHIJAAAA	angermod{0.6} [AngerMod - Angry neutral mobs] (AngerMod-1.7.10-0.6.jar) 
	UCHIJAAAA	AppleCore{1.3.9} [AppleCore] (AppleCore-mc1.7.10-1.3.9.jar) 
	UCHIJAAAA	aquacreeper{1.0.1} [Aqua Creeper] (AquaCreepers-1.0.1.jar) 
	UCHIJAAAA	ArchitectureCraft{1.7.2} [ArchitectureCraft] (ArchitectureCraft-1.7.3-mc1.7.10.jar) 
	UCHIJAAAA	CoFHCore{1.7.10R3.1.4} [CoFH Core] (CoFHCore-[1.7.10]3.1.4-329.jar) 
	UCHIJAAAA	asielib{0.4.9} [asielib] (AsieLib-1.7.10-0.4.9.jar) 
	UCHIJAAAA	Thaumcraft{4.2.3.5} [Thaumcraft] (Thaumcraft-1.7.10-4.2.3.5.jar) 
	UCHIJAAAA	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) 
	UCHIJAAAA	Automagy{0.28.2} [Automagy] (Automagy-1.7.10-0.28.2.jar) 
	UCHIJAAAA	AWWayofTime{v1.3.3} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.7.10-1.3.3-29.jar) 
	UCHIJAAAA	Avaritia{1.17} [Avaritia] (Avaritia-1.17.jar) 
	UCHIJAAAA	wanionlib{1.7.10-1.8} [WanionLib] (WanionLib-1.7.10-1.8.jar) 
	UCHIJAAAA	avaritiaddons{1.4b} [Avaritiaddons] (Avaritiaddons-1.4b.jar) 
	UCHIJAAAA	Backpack{2.2.0} [Backpack] (backpack-2.2.4-1.7.x.jar) 
	UCHIJAAAA	BanHammerJ{0.0.1} [BanHammerJ] (BanHammerJ-0.0.1.jar) 
	UCHIJAAAA	Mantle{1.7.10-0.3.2.jenkins191} [Mantle] (Mantle-1.7.10-0.3.2b.jar) 
	UCHIJAAAA	Natura{2.2.0} [Natura] (natura-1.7.10-126.240ee6f.jar) 
	UCHIJAAAA	BiomesOPlenty{2.1.0} [Biomes O' Plenty] (BiomesOPlenty-1.7.10-2.1.0.2309-universal.jar) 
	UCHIJAAAA	HardcoreEnderExpansion{1.8.6} [Hardcore Ender Expansion] (HardcoreEnderExpansion  MC-1.7.10  v1.8.6.jar) 
	UCHIJAAAA	Forestry{4.2.21.0} [Forestry for Minecraft] (forestry_1.7.10-4.2.21.0.jar) 
	UCHIJAAAA	BinnieCore{2.0.26} [Binnie Core] (binnie-mods-1.7.10-gtnh-2.0.26.jar) 
	UCHIJAAAA	ExtraTrees{2.0.26} [Extra Trees] (binnie-mods-1.7.10-gtnh-2.0.26.jar) 
	UCHIJAAAA	BuildCraft|Core{7.1.23} [BuildCraft] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Energy{7.1.23} [BC Energy] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	ForgeMultipart{1.2.0.347} [Forge Multipart] (ForgeMultipart-1.7.10-1.2.0.347-universal.jar) 
	UCHIJAAAA	ExtraUtilities{1.2.12} [Extra Utilities] (extrautilities-1.2.13a.jar) 
	UCHIJAAAA	TConstruct{1.7.10-1.8.9.14-GTNH.DEV.0605dd63} [Tinkers' Construct] (TConstruct-1.7.10-1.8.9.14-GTNH.jar) 
	UCHIJAAAA	GalacticraftCore{3.0.12} [Galacticraft Core] (GalacticraftCore-1.7-3.0.12.504.jar) 
	UCHIJAAAA	BuildCraft|Transport{7.1.23} [BC Transport] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Factory{7.1.23} [BC Factory] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Silicon{7.1.23} [BC Silicon] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Builders{7.1.23} [BC Builders] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	GalacticraftMars{3.0.12} [Galacticraft Planets] (Galacticraft-Planets-1.7-3.0.12.504.jar) 
	UCHIJAAAA	endercore{1.7.10-0.2.0.39_beta} [EnderCore] (EnderCore-1.7.10-0.2.0.39_beta.jar) 
	UCHIJAAAA	Railcraft{9.12.2.1} [Railcraft] (Railcraft_1.7.10-9.12.2.1.jar) 
	UCHIJAAAA	TwilightForest{2.3.8dev} [The Twilight Forest] (twilightforest-1.7.10-2.3.8dev.jar) 
	UCHIJAAAA	chisel{release-CUSTOM-d8fe69a} [Chisel] (Chisel-release-CUSTOM-d8fe69a.jar) 
	UCHIJAAAA	EnderIO{1.7.10-2.3.0.434_beta.0_master_a833d33} [Ender IO] (EnderIO-1.7.10-2.3.0.434_beta.jar) 
	UCHIJAAAA	supersolarpanel{1.7.10-1.1.1-GT-NH-Mod} [Super Solar Panel] (SSP-1.1.2-GT-NH-Mod.jar) 
	UCHIJAAAA	dreamcraft{1.6.31} [GT: New Horizons] (GTNewHorizonsCoreMod-1.7.10-1.6.31.jar) 
	UCHIJAAAA	harvestcraft{1.7.10j} [Pam's HarvestCraft] (Pam's Harvestcraft 1.7.10n.jar) 
	UCHIJAAAA	Translocator{1.1.2.16} [Translocator] (Translocator-1.7.10-1.1.2.16-universal.jar) 
	UCHIJAAAA	gregtech{MC1710} [GregTech] (gregtech-5.09.33.31.jar) 
	UCHIJAAAA	DummyCore{1.13} [DummyCore] (DummyCore1.13.jar) 
	UCHIJAAAA	thaumicbases{1.4.20} [Thaumic Bases] (Thaumic-Based- 1.4.20-0.jar) 
	UCHIJAAAA	witchery{0.24.1} [Witchery] (witchery-1.7.10-0.24.1.jar) 
	UCHIJAAAA	Ztones{1.7.10} [Ztones] (Ztones-1.7.10-2.2.2.jar) 
	UCHIJAAAA	croploadcore{0.1.4} [CropLoadCore] (CropLoadCore-0.1.4.jar) 
	UCHIJAAAA	berriespp{1.3.5.1} [Crops++] (CropsPP-1.3.5.1.jar) 
	UCHIJAAAA	bartworks{0.5.10} [BartWorks] (bartworks.1.7.10.-0.5.10.jar) 
	UCHIJAAAA	ThaumicTinkerer{2.5} [Thaumic Tinkerer] (ThaumicTinkerer-1.7.10-2.5-557-GTNH.jar) 
	UCHIJAAAA	ForbiddenMagic{1.7.10-0.575} [Forbidden Magic] (Forbidden Magic-1.7.10-0.575.jar) 
	UCHIJAAAA	ExtraBees{2.0.26} [Extra Bees] (binnie-mods-1.7.10-gtnh-2.0.26.jar) 
	UCHIJAAAA	MagicBees{2.4.3} [Magic Bees] (magicbees-1.7.10-2.4.3.jar) 
	UCHIJAAAA	EMT{1.2.7.7} [Electro-Magic Tools] (ElectroMagicTools-1.2.7.7[1.7.10].jar) 
	UCHIJAAAA	IC2NuclearControl{2.4.3a} [Nuclear Control 2] (IC2NuclearControl-2.4.3a.jar) 
	UCHIJAAAA	OpenMods{0.10} [OpenMods] (OpenModsLib-1.7.10-0.10-snapshot-465.jar) 
	UCHIJAAAA	OpenBlocks{1.6} [OpenBlocks] (OpenBlocks-1.7.10-1.6-snapshot-753.jar) 
	UCHIJAAAA	StevesCarts{2.0.0.b18} [Steve's Carts 2] (StevesCarts2.0.0.b18.jar) 
	UCHIJAAAA	tectech{GRADLETOKEN_VERSION} [TecTech] (TecTech-1.7.10-3.7.3.jar) 
	UCHIJAAAA	TGregworks{1.0.18} [Tinkers' Gregworks] (TGregworks-1.7.10-1.0.18.jar) 
	UCHIJAAAA	miscutils{1.7.03.45} [GT++] (GT-PlusPlus-1.7.04.483.jar) 
	UCHIJAAAA	bartworkscrossmod{0.5.10} [BartWorks Mod Additions] (bartworks.1.7.10.-0.5.10.jar) 
	UCHIJAAAA	betterbuilderswands{0.8.1} [Better Builder's Wands] (BetterBuildersWands-0.8.1-1.7.10r92+aec06c3.jar) 
	UCHIJAAAA	betterquesting{3.0.322} [Better Questing] (BetterQuesting-3.0.322.jar) 
	UCHIJAAAA	BiblioCraft{1.11.7} [BiblioCraft] (BiblioCraft[v1.11.7][MC1.7.10].jar) 
	UCHIJAAAA	BiblioWoodsBoP{1.9} [BiblioWoods Biomes O'Plenty Edition] (BiblioWoods[BiomesOPlenty][v1.9].jar) 
	UCHIJAAAA	BiblioWoodsForestry{1.7} [BiblioWoods Forestry Edition] (BiblioWoods[Forestry][v1.7].jar) 
	UCHIJAAAA	BiblioWoodsNatura{1.5} [BiblioWoods Natura Edition] (BiblioWoods[Natura][v1.5].jar) 
	UCHIJAAAA	Botany{2.0.26} [Botany] (binnie-mods-1.7.10-gtnh-2.0.26.jar) 
	UCHIJAAAA	Genetics{2.0.26} [Genetics] (binnie-mods-1.7.10-gtnh-2.0.26.jar) 
	UCHIJAAAA	blocklimiter{0.52} [BlockLimiter. Restict your Blocks] (BlockLimiter-1.7.10-0.52.jar) 
	UCHIJAAAA	BloodArsenal{1.2-8} [Blood Arsenal] (BloodArsenal-1.7.10-1.2-8.jar) 
	UCHIJAAAA	BrandonsCore{1.0.0.12} [Brandon's Core] (BrandonsCore-1.0.0.12.jar) 
	UCHIJAAAA	BuildCraft|Robotics{7.1.23} [BC Robotics] (buildcraft-7.1.23.jar) 
	UCHIJAAAA	BuildCraft|Compat{7.1.7} [BuildCraft Compat] (buildcraft-compat-7.1.7.jar) 
	UCHIJAAAA	OilTweak{1.0.3} [BuildCraft Oil Tweak] (BuildCraftOilTweak-1.7.10-1.0.3.jar) 
	UCHIJAAAA	CarpentersBlocks{3.3.8.1} [Carpenter's Blocks] (Carpenter's Blocks v3.3.8.1 - MC 1.7.10.jar) 
	UCHIJAAAA	catwalks{2.0.4} [Catwalks Mod] (catwalks-2.0.4.jar) 
	UCHIJAAAA	catwalkmod{x.x.x} [Legacy Catwalk Compat] (catwalks-2.0.4.jar) 
	UCHIJAAAA	cb4bq{1} [Command Blocks for Better Questing] (CB4BQ-3.jar) 
	UCHIJAAAA	chiseltones{@VERSION@} [Chisel Tones] (ChiselTones-1.7.10-1.0-3.jar) 
	UCHIJAAAA	compactkineticgenerators{1.7.10-1.0} [Compact Kinetic Wind and Water Generators] (CompactKineticGenerators-1.7.10-1.0.jar) 
	UCHIJAAAA	EnderStorage{1.4.7.38} [EnderStorage] (EnderStorage-1.7.10-1.4.7.38-universal.jar) 
	UCHIJAAAA	MrTJPCoreMod{1.1.0.33} [MrTJPCore] (MrTJPCore-1.1.0.33-universal.jar) 
	UCHIJAAAA	ProjRed|Core{4.7.0pre11.52} [ProjectRed Core] (ProjectRed-1.7.10-4.7.0pre12.95-Base.jar) 
	UCHIJAAAA	ProjRed|Transmission{4.7.0pre11.52} [ProjectRed Transmission] (ProjectRed-1.7.10-4.7.0pre12.95-Integration.jar) 
	UCHIJAAAA	OpenComputers{1.7.5.1+4f95ff4} [OpenComputers] (OpenComputers-MC1.7.10-1.7.5.1-universal.jar) 
	UCHIJAAAA	computronics{1.6.6} [Computronics] (Computronics-1.7.10-1.6.6.jar) 
	UCHIJAAAA	MineTweaker3{3.0.12} [MineTweaker 3] (CraftTweaker-1.7.10-3.0.12.jar) 
	UCHIJAAAA	DraconicEvolution{1.0.2h} [Draconic Evolution] (Draconic-Evolution-1.7.10-1.0.2n.jar) 
	UCHIJAAAA	EnderZoo{1.7.10-1.0.16.0} [Ender Zoo] (EnderZoo-1.7.10-1.0.16.0.jar) 
	UCHIJAAAA	enhancedlootbags{1.0.2a} [Enhanced LootBags] (EnhancedLootBags-1.7.10-1.0.2b.jar) 
	UCHIJAAAA	enhancedportals{3.0.12} [EnhancedPortals] (EnhancedPortals_1.7.10-universal-3.0.12.jar) 
	UCHIJAAAA	universalsingularities{1.7.10-8.5a} [Universal Singularities] (Universal Singularities-1.7.10-8.6.jar) 
	UCHIJAAAA	eternalsingularity{1.7.10-.1.0.3} [Eternal Singularity] (EternalSingularity-1.7.10-1.0.4.jar) 
	UCHIJAAAA	extracells{2.4.2} [Extra Cells 2] (ExtraCells-1.7.10-2.4.2bNIGHTLY.jar) 
	UCHIJAAAA	findit{1.0.1} [FindIt] (findit-1.7.10-1.0.1.jar) 
	UCHIJAAAA	FloodLights{1.2.8-137} [Flood Lights] (FloodLights-1.7.10-1.2.8-137.jar) 
	UCHIJAAAA	gadomancy{1.0.7.4} [Gadomancy] (gadomancy-1.7.10-1.0.7.6.jar) 
	UCHIJAAAA	galacticgreg{1.0.2} [Galactic Greg] (GalacticGreg-1.7.10-1.0.2.jar) 
	UCHIJAAAA	GalaxySpace{1.1.3} [GalaxySpace] (GalaxySpace-1.1.3c_STABLE.jar) 
	UCHIJAAAA	gendustry{1.6.4.135} [GenDustry] (gendustry-1.6.4.135-mc1.7.10.jar) 
	UCHIJAAAA	GraviSuite{2.0.68u} [Graviation Suite] (GraviSuite-1.7.10-2.0.68u.jar) 
	UCHIJAAAA	detravscannermod{1.5.6} [GT Scanner Mod] (GT Scanner Mod-1.5.6a.jar) 
	UCHIJAAAA	ToxicEverglades{0.1} [GT++ Toxic Everglades] (GT-PlusPlus-1.7.04.483.jar) 
	UCHIJAAAA	Australia{0.1} [GT++ Australia] (GT-PlusPlus-1.7.04.483.jar) 
	UCHIJAAAA	TaintedMagic{r7.6} [Tainted Magic] (Tainted-Magic-r7.6.jar) 
	UCHIJAAAA	ThaumicExploration{1.1.73} [Thaumic Exploration] (Thaumic-Exploration-[1.7.10] 1.1.73-0.jar) 
	UCHIJAAAA	gtnhtcwands{1.2.0} [GTNH-TC-Wands] (GTNH-TC-Wands-1.2.0.jar) 
	UCHIJAAAA	GTTweaker{1.0} [GTTweaker] (GTTweaker-1.7.10-1.7.5.jar) 
	UCHIJAAAA	HardcoreDarkness{1.7} [Hardcore Darkness] (HardcoreDarkness-MC1.7.10-1.7.jar) 
	UCHIJAAAA	HoloInventory{1.9.1.124} [HoloInventory] (HoloInventory-1.7.10-1.9.1.124.jar) 
	UCHIJAAAA	ifu{1.5} [I Will Find You] (IFU-1.6.jar) 
	UCHIJAAAA	InfernalMobs{1.6.6} [Infernal Mobs] (InfernalMobs-1.7.10.1.6.6.jar) 
	UCHIJAAAA	inpure|core{1.7.10R1.0.0B9} [INpureCore] (INpureCore-[1.7.10]1.0.0B9-62.jar) 
	UCHIJAAAA	IronChest{6.0.62.742} [Iron Chest] (ironchest-1.7.10-6.0.62.742-universal.jar) 
	UCHIJAAAA	ironchestminecarts{1.0.4} [Iron Chest Minecarts] (IronChestMinecarts-1.0.4.jar) 
	UCHIJAAAA	irontank{@VERSION@ build @BUILD@} [Iron Tanks] (irontanks-1.1.16.jar) 
	UCHIJAAAA	JABBA{1.2.2} [JABBA] (Jabba-1.2.5_1.7.10.jar) 
	UCHIJAAAA	JourneyMapServer{1.0.5_MC1.7.10} [JourneyMapServer] (JourneyMapServer1.0.5_MC1.7.10.jar) 
	UCHIJAAAA	kekztech{0.2.2} [KekzTech] (kekztech-0.2.2.jar) 
	UCHIJAAAA	lootgames{1.0.18} [LootGames] (LootGames-1.7.10-1.0.18.jar) 
	UCHIJAAAA	malisiscore{1.7.10-0.14.3} [MalisisCore] (malisiscore-1.7.10-0.14.3.jar) 
	UCHIJAAAA	malisisdoors{1.7.10-1.13.2} [Malisis' Doors] (malisisdoors-1.7.10-1.13.2.jar) 
	UCHIJAAAA	modtweaker2{0.9.6} [Mod Tweaker 2] (ModTweaker2-0.9.6.173.jar) 
	UCHIJAAAA	Morpheus{1.7.10-1.6.21} [Morpheus] (Morpheus-1.7.10-1.6.21.jar) 
	UCHIJAAAA	naturescompass{1.3.1} [Nature's Compass] (NaturesCompass-1.7.10-1.3.1.jar) 
	UCHIJAAAA	NEIAddons{1.12.15.41} [NEI Addons] (neiaddons-1.12.15.41-mc1.7.10.jar) 
	UCHIJAAAA	NEIAddons|Developer{1.12.15.41} [NEI Addons: Developer Tools] (neiaddons-1.12.15.41-mc1.7.10.jar) 
	UCHIJAAAA	NEIAddons|AppEng{1.12.15.41} [NEI Addons: Applied Energistics 2] (neiaddons-1.12.15.41-mc1.7.10.jar) 
	UCHIJAAAA	NEIAddons|Botany{1.12.15.41} [NEI Addons: Botany] (neiaddons-1.12.15.41-mc1.7.10.jar) 
	UCHIJAAAA	NEIAddons|Forestry{1.12.15.41} [NEI Addons: Forestry] (neiaddons-1.12.15.41-mc1.7.10.jar) 
	UCHIJAAAA	NEIAddons|CraftingTables{1.12.15.41} [NEI Addons: Crafting Tables] (neiaddons-1.12.15.41-mc1.7.10.jar) 
	UCHIJAAAA	NEIAddons|ExNihilo{1.12.15.41} [NEI Addons: Ex Nihilo] (neiaddons-1.12.15.41-mc1.7.10.jar) 
	UCHIJAAAA	neiintegration{1.1.2} [NEI Integration] (NEIIntegration-MC1.7.10-1.1.2.jar) 
	UCHIJAAAA	NodalMechanics{1.1.0} [NodalMechanics] (NodalMechanics-.1.7.10.-1.1.0-K.jar) 
	UCHIJAAAA	openglasses{1.0.46} [OC Glasses] (OpenGlasses-1.0.46.jar) 
	UCHIJAAAA	openmodularturrets{2.2.11-238} [Open Modular Turrets] (OpenModularTurrets-1.7.10-2.2.11-238.jar) 
	UCHIJAAAA	openprinter{0.1.0.132} [OpenPrinter] (OpenPrinter-MC1.7.10-0.1.0.132.jar) 
	UCHIJAAAA	opensecurity{1.0.117} [OpenSecurity] (OpenSecurity-1.7.10-1.0-117.jar) 
	UCHIJAAAA	p455w0rdslib{1.0.4} [p455w0rd's Library] (p455w0rdslib-1.7.10-1.0.4.jar) 
	UCHIJAAAA	harvestthenether{1.7.10} [Pam's Harvest the Nether] (Pam's Harvest the Nether 1.7.10a.jar) 
	UCHIJAAAA	ProjectBlue{1.1.6} [Project Blue] (ProjectBlue-1.1.6-mc1.7.10.jar) 
	UCHIJAAAA	ProjRed|Transportation{4.7.0pre11.52} [ProjectRed Transportation] (ProjectRed-1.7.10-4.7.0pre12.95-Mechanical.jar) 
	UCHIJAAAA	ProjRed|Exploration{4.7.0pre11.52} [ProjectRed Exploration] (ProjectRed-1.7.10-4.7.0pre12.95-World.jar) 
	UCHIJAAAA	ProjRed|Compatibility{4.7.0pre11.52} [ProjectRed Compatibility] (ProjectRed-1.7.10-4.7.0pre12.95-Compat.jar) 
	UCHIJAAAA	ProjRed|Integration{4.7.0pre11.52} [ProjectRed Integration] (ProjectRed-1.7.10-4.7.0pre12.95-Integration.jar) 
	UCHIJAAAA	ProjRed|Fabrication{4.7.0pre11.52} [ProjectRed Fabrication] (ProjectRed-1.7.10-4.7.0pre12.95-Fabrication.jar) 
	UCHIJAAAA	ProjRed|Illumination{4.7.0pre11.52} [ProjectRed Illumination] (ProjectRed-1.7.10-4.7.0pre12.95-Lighting.jar) 
	UCHIJAAAA	ProjRed|Expansion{4.7.0pre11.52} [ProjectRed Expansion] (ProjectRed-1.7.10-4.7.0pre12.95-Mechanical.jar) 
	UCHIJAAAA	questbook{2.1.1-1.7.10} [Better Questing Quest Book] (questbook-2.1.1-1.7.10.jar) 
	UCHIJAAAA	RandomThings{2.2.4} [Random Things] (RandomThings-2.2.4.jar) 
	UCHIJAAAA	RIO{2.4.0} [Remote IO] (RemoteIO-1.7.10-2.4.0-universal.jar) 
	UCHIJAAAA	bq_rf{3.0.29} [RF Expansion] (RFExpansion-3.0.29.jar) 
	UCHIJAAAA	Roguelike{1.5.0} [Roguelike Dungeons] (roguelike-1.7.10-1.5.0b.jar) 
	UCHIJAAAA	RWG{Alpha 1.3.2} [Realistic World Gen Alpha] (RWG-alpha-1.3.2.jar) 
	UCHIJAAAA	SGCraft{1.13.3} [Greg's SG Craft] (SGCraft-1.13.3-mc1.7.10.jar) 
	UCHIJAAAA	sleepingbag{0.1.1} [Sleeping Bag] (SleepingBag-1.7.10-0.1.1.jar) 
	UCHIJAAAA	SpecialMobs{GRADLETOKEN_VERSION} [Special Mobs] (SpecialMobs-1.7.10-3.3.3.jar) 
	UCHIJAAAA	SpiceOfLife{unspecified} [The Spice of Life] (SpiceOfLife-mc1.7.10-1.3.12.jar) 
	UCHIJAAAA	bq_standard{3.0.174} [Standard Expansion] (StandardExpansion-3.0.174.jar) 
	UCHIJAAAA	StevesFactoryManager{A92} [Steve's Factory Manager] (StevesFactoryManager-A94-GTNH.jar) 
	UCHIJAAAA	StevesAddons{0.10.16} [Steve's Addons] (StevesAddons-1.7.10-0.10.16.jar) 
	UCHIJAAAA	StorageDrawers{1.7.10-1.11.1-gtnh} [Storage Drawers] (StorageDrawers-1.7.10-1.11.1-gtnh.jar) 
	UCHIJAAAA	StorageDrawersBop{1.7.10-1.1.1b} [Storage Drawers: Biomes O' Plenty Pack] (StorageDrawers-BiomesOPlenty-1.7.10-1.1.1c.jar) 
	UCHIJAAAA	StorageDrawersForestry{1.7.10-1.1.2b} [Storage Drawers: Forestry Pack] (StorageDrawers-Forestry-1.7.10-1.1.2c.jar) 
	UCHIJAAAA	StorageDrawersMisc{1.7.10-1.1.2b} [Storage Drawers: Misc Pack] (StorageDrawers-Misc-1.7.10-1.1.2c.jar) 
	UCHIJAAAA	StorageDrawersNatura{1.7.10-1.1.1b} [Storage Drawers: Natura Pack] (StorageDrawers-Natura-1.7.10-1.1.1c.jar) 
	UCHIJAAAA	SuperTic{1.2.1} [SuperTic] (SuperTiC-1.2.1.jar) 
	UCHIJAAAA	tcinventoryscan{1.0.11} [TC Inventory Scanning] (tcinventoryscan-mc1.7.10-1.0.11.jar) 
	UCHIJAAAA	ThaumcraftMobAspects{1.7.x-2B} [Thaumcraft Mob Aspects] (ThaumcraftMobAspects-1.7.x-2B-snapshot-17.jar) 
	UCHIJAAAA	thaumcraftneiplugin{@VERSION@} [Thaumcraft NEI Plugin] (thaumcraftneiplugin-1.7.10-1.7a.jar) 
	UCHIJAAAA	ThaumicMachina{0.2.1} [Thaumic Machina] (Thaumic Machina-1.7.10-0.2.1.jar) 
	UCHIJAAAA	thaumicenergistics{1.1.3.0} [Thaumic Energistics] (thaumicenergistics-1.1.3.0.jar) 
	UCHIJAAAA	ThaumicHorizons{1.1.9.15} [Thaumic Horizons] (ThaumicHorizons-1.7.10 1.1.9.15.jar) 
	UCHIJAAAA	TiCTooltips{1.2.5} [TiC Tooltips] (TiCTooltips-mc1.7.10-1.2.5.jar) 
	UCHIJAAAA	tinkersdefense{1.2} [Tinkers' Defense] (Tinkers-Defense-1.2.1d.jar) 
	UCHIJAAAA	TMechworks{0.2.15.106} [Tinkers' Mechworks] (TMechworks-1.7.10-0.2.15.106.jar) 
	UCHIJAAAA	TML{4.0.3} [TooMuchLoot] (TooMuchLoot-1.7.10-4.0.3.B2.jar) 
	UCHIJAAAA	TravellersGear{1.16.6} [Traveller's Gear] (TravellersGear-1.7.10-1.16.6a.jar) 
	UCHIJAAAA	utilityworlds{1.0.9d} [Utility Worlds] (utilityworlds-1.0.9e.jar) 
	UCHIJAAAA	WailaHarvestability{1.1.6} [Waila Harvestability] (WailaHarvestability-mc1.7.10-1.1.6.jar) 
	UCHIJAAAA	wailaplugins{MC1.7.10-0.2.3-null} [WAILA Plugins] (WAILAPlugins-MC1.7.10-0.2.3.jar) 
	UCHIJAAAA	WarpTheory{1.7.10R1.0f.Custom} [WarpTheory] (WarpTheory-1.7.10-1.0k-Dev.jar) 
	UCHIJAAAA	wawla{1.3.1} [What Are We Looking At] (Wawla-1.0.5.120.jar) 
	UCHIJAAAA	ae2wct{1.7.10-rv3-1.8.7.4-bigbass-1} [AE2 Wireless Crafting Terminal] (WirelessCraftingTerminal-1.7.10-rv3-1.8.7.4-bigbass-1.jar) 
	UCHIJAAAA	WitchingGadgets{1.2.10-GTNH} [Witching Gadgets] (WitchingGadgets-1.7.10-1.2.10-GTNH.jar) 
	UCHIJAAAA	WR-CBE|Core{1.4.2-GTNH} [WR-CBE Core] (WR-CBE-1.7.10-1.4.2-GTNH-universal.jar) 
	UCHIJAAAA	WR-CBE|Addons{1.4.2-GTNH} [WR-CBE Addons] (WR-CBE-1.7.10-1.4.2-GTNH-universal.jar) 
	UCHIJAAAA	WR-CBE|Logic{1.4.2-GTNH} [WR-CBE Logic] (WR-CBE-1.7.10-1.4.2-GTNH-universal.jar) 
	UCHIJAAAA	ForgeMicroblock{1.2.0.347} [Forge Microblocks] (ForgeMultipart-1.7.10-1.2.0.347-universal.jar) 
	UCHIJAAAA	McMultipart{1.2.0.347} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.2.0.347-universal.jar) 
	UCHIJAAAA	ForgeRelocation{0.0.1.4} [ForgeRelocation] (ForgeRelocation-0.0.1.4-universal.jar) 
	UCHIJAAAA	MCFrames{1.0} [MCFrames] (ForgeRelocation-0.0.1.4-universal.jar) 
	UCHIJAAAA	RelocationFMP{0.0.1.2} [RelocationFMP] (ForgeRelocationFMP-0.0.1.2-universal.jar) 
	UCHIJAAAA	HungerOverhaul{1.7.10-1.0.2.DEV.80bb6db} [Hunger Overhaul] (HungerOverhaul-1.7.10-1.0.2.jar) 
	UCHIJAAAA	IguanaTweaksTConstruct{1.7.10-2.1.6.2} [Iguana Tinker Tweaks] (IguanaTinkerTweaks-1.7.10-2.1.6.2.jar) 
	OpenModsLib class transformers: [stencil_patches:ENABLED],[movement_callback:ENABLED],[player_damage_hook:FINISHED],[map_gen_fix:FINISHED],[gl_capabilities_hook:ENABLED],[player_render_hook:ENABLED]
	Class transformer null safety: all safe
	AE2 Version: beta rv3-beta-22 for Forge 10.13.4.1448
	CoFHCore: -[1.7.10]3.1.4-329
	Mantle Environment: Environment healthy.
	TConstruct Environment: Environment healthy.
	List of loaded APIs: 
		* ae2wct|API (1.7.10-rv3-1.8.6b) from WirelessCraftingTerminal-1.7.10-rv3-1.8.7.4-bigbass-1.jar
		* AppleCoreAPI (1.2.0) from AppleCore-mc1.7.10-1.3.9.jar
		* appliedenergistics2|API (rv3) from appliedenergistics2-rv3-beta-22.jar
		* asielibAPI (1.1) from AsieLib-1.7.10-0.4.9.jar
		* asielibAPI|chat (1.0) from AsieLib-1.7.10-0.4.9.jar
		* asielibAPI|tile (1.0) from AsieLib-1.7.10-0.4.9.jar
		* asielibAPI|tool (1.1) from AsieLib-1.7.10-0.4.9.jar
		* BattlePlayer (0.1) from 1.7.10-MB_Battlegear2-Bullseye-1.0.8.5.jar
		* Baubles|API (1.0.1.10) from Baubles-1.7.10-1.0.1.11.jar
		* BetterQuesting|API (3.2) from BetterQuesting-3.0.322.jar
		* BetterQuesting|API2 (3.1) from BetterQuesting-3.0.322.jar
		* BiomesOPlentyAPI (1.0.0) from BiomesOPlenty-1.7.10-2.1.0.2309-universal.jar
		* BloodMagicAPI (1.3.3-13) from BloodMagic-1.7.10-1.3.3-29.jar
		* BotaniaAPI (61) from Avaritia-1.17.jar
		* BuildCraftAPI|blocks (1.0) from buildcraft-7.1.23.jar
		* BuildCraftAPI|blueprints (1.5) from buildcraft-7.1.23.jar
		* BuildCraftAPI|boards (2.0) from buildcraft-7.1.23.jar
		* BuildCraftAPI|core (1.0) from buildcraft-7.1.23.jar
		* BuildCraftAPI|crops (1.1) from Railcraft_1.7.10-9.12.2.1.jar
		* BuildCraftAPI|events (2.0) from buildcraft-7.1.23.jar
		* BuildCraftAPI|facades (1.1) from buildcraft-7.1.23.jar
		* BuildCraftAPI|filler (4.0) from buildcraft-7.1.23.jar
		* BuildCraftAPI|fuels (2.0) from buildcraft-7.1.23.jar
		* BuildCraftAPI|gates (4.1) from buildcraft-7.1.23.jar
		* BuildCraftAPI|items (1.1) from Railcraft_1.7.10-9.12.2.1.jar
		* BuildCraftAPI|library (2.0) from Railcraft_1.7.10-9.12.2.1.jar
		* BuildCraftAPI|lists (1.0) from buildcraft-7.1.23.jar
		* BuildCraftAPI|power (1.3) from buildcraft-7.1.23.jar
		* BuildCraftAPI|recipes (3.1) from buildcraft-7.1.23.jar
		* BuildCraftAPI|robotics (3.0) from buildcraft-7.1.23.jar
		* BuildCraftAPI|statements (1.1) from Railcraft_1.7.10-9.12.2.1.jar
		* BuildCraftAPI|tablet (1.0) from buildcraft-7.1.23.jar
		* BuildCraftAPI|tiles (1.2) from Railcraft_1.7.10-9.12.2.1.jar
		* BuildCraftAPI|tools (1.0) from Railcraft_1.7.10-9.12.2.1.jar
		* BuildCraftAPI|transport (4.1) from buildcraft-7.1.23.jar
		* CarpentersBlocks|API (3.3.7) from Carpenter's Blocks v3.3.8.1 - MC 1.7.10.jar
		* ChiselAPI (0.1.1) from Chisel-release-CUSTOM-d8fe69a.jar
		* ChiselAPI|Carving (0.1.1) from Chisel-release-CUSTOM-d8fe69a.jar
		* ChiselAPI|Rendering (0.1.1) from Chisel-release-CUSTOM-d8fe69a.jar
		* CoFHAPI (1.7.10R1.0.2) from RFExpansion-3.0.29.jar
		* CoFHAPI|block (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHAPI|core (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|energy (1.7.10R1.0.2) from p455w0rdslib-1.7.10-1.0.4.jar
		* CoFHAPI|fluid (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHAPI|inventory (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|item (1.7.10R1.0.13) from EnderIO-1.7.10-2.3.0.434_beta.jar
		* CoFHAPI|modhelpers (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|tileentity (1.7.10R1.0.13) from EnderIO-1.7.10-2.3.0.434_beta.jar
		* CoFHAPI|transport (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHAPI|world (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|audio (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|gui (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|container (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|element (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|element|listbox (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|gui|slot (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|inventory (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|render (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|render|particle (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util|helpers (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|util|position (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|world (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|world|feature (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* computronicsAPI (1.3) from Computronics-1.7.10-1.6.6.jar
		* computronicsAPI|audio (1.0) from Computronics-1.7.10-1.6.6.jar
		* computronicsAPI|chat (1.3) from Computronics-1.7.10-1.6.6.jar
		* computronicsAPI|multiperipheral (1.1) from Computronics-1.7.10-1.6.6.jar
		* computronicsAPI|tape (1.0) from Computronics-1.7.10-1.6.6.jar
		* DraconicEvolution|API (1.2) from Draconic-Evolution-1.7.10-1.0.2n.jar
		* DualWield (0.1) from 1.7.10-MB_Battlegear2-Bullseye-1.0.8.5.jar
		* EnderIOAPI (0.0.2) from EnderIO-1.7.10-2.3.0.434_beta.jar
		* EnderIOAPI|Redstone (0.0.2) from EnderIO-1.7.10-2.3.0.434_beta.jar
		* EnderIOAPI|Teleport (0.0.2) from EnderIO-1.7.10-2.3.0.434_beta.jar
		* EnderIOAPI|Tools (0.0.2) from EnderIO-1.7.10-2.3.0.434_beta.jar
		* ForestryAPI|apiculture (4.8.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|arboriculture (4.2.1) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|circuits (3.1.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|core (5.0.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|farming (2.1.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|food (1.1.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|fuels (2.0.1) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|genetics (4.7.1) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|hives (4.1.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|lepidopterology (1.3.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|mail (3.0.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|multiblock (3.0.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|recipes (5.4.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|storage (3.0.0) from forestry_1.7.10-4.2.21.0.jar
		* ForestryAPI|world (2.1.0) from forestry_1.7.10-4.2.21.0.jar
		* ForgeRelocation|API (0.0.1.4) from ForgeRelocation-0.0.1.4-universal.jar
		* Galacticraft API (1.1) from GalacticraftCore-1.7-3.0.12.504.jar
		* gendustryAPI (2.3.0) from gendustry-1.6.4.135-mc1.7.10.jar
		* Heraldry (alpha) from 1.7.10-MB_Battlegear2-Bullseye-1.0.8.5.jar
		* IC2API (1.0) from industrialcraft-2-2.2.828a-experimental.jar
		* inpure|api (1.7) from INpureCore-[1.7.10]1.0.0B9-62.jar
		* NuclearControlAPI (v1.0.5) from IC2NuclearControl-2.4.3a.jar
		* OilTweakAPI (1.0.0) from BuildCraftOilTweak-1.7.10-1.0.3.jar
		* OilTweakAPI|blacklist (1.0.0) from BuildCraftOilTweak-1.7.10-1.0.3.jar
		* OpenBlocks|API (1.1) from OpenBlocks-1.7.10-1.6-snapshot-753.jar
		* OpenComputersAPI|Component (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|Core (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|Driver (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|Driver|Item (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|Event (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|FileSystem (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|Internal (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|Machine (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|Manual (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|Network (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* OpenComputersAPI|Prefab (6.0.0-alpha) from OpenComputers-MC1.7.10-1.7.5.1-universal.jar
		* Quiver (0.2) from 1.7.10-MB_Battlegear2-Bullseye-1.0.8.5.jar
		* RailcraftAPI|bore (1.0.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|carts (1.6.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|core (1.5.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|crafting (1.0.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|electricity (2.0.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|events (1.0.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|fuel (1.0.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|helpers (1.1.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|items (1.0.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|locomotive (1.1.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|signals (2.3.0) from Railcraft_1.7.10-9.12.2.1.jar
		* RailcraftAPI|tracks (2.3.0) from Railcraft_1.7.10-9.12.2.1.jar
		* Shield (0.1) from 1.7.10-MB_Battlegear2-Bullseye-1.0.8.5.jar
		* StorageDrawersAPI (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.11.1-gtnh.jar
		* StorageDrawersAPI|config (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.11.1-gtnh.jar
		* StorageDrawersAPI|event (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.11.1-gtnh.jar
		* StorageDrawersAPI|inventory (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.11.1-gtnh.jar
		* StorageDrawersAPI|pack (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.11.1-gtnh.jar
		* StorageDrawersAPI|registry (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.11.1-gtnh.jar
		* StorageDrawersAPI|render (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.11.1-gtnh.jar
		* StorageDrawersAPI|storage (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.11.1-gtnh.jar
		* StorageDrawersAPI|storage-attribute (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.11.1-gtnh.jar
		* Thaumcraft|API (4.2.2.0) from Railcraft_1.7.10-9.12.2.1.jar
		* thaumicenergistics|API (1.1) from thaumicenergistics-1.1.3.0.jar
		* TravellersGear|API (1.0) from TravellersGear-1.7.10-1.16.6a.jar
		* WailaAPI (1.2) from Waila-1.5.10_1.7.10.jar
		* Weapons (0.1) from 1.7.10-MB_Battlegear2-Bullseye-1.0.8.5.jar
	Chisel: Errors like "[FML]: Unable to lookup ..." are NOT the cause of this crash. You can safely ignore these errors. And update forge while you're at it.
	EnderIO: Found the following problem(s) with your installation:
                  * An unknown AE2 API is installed (rv3 from appliedenergistics2-rv3-beta-22.jar).
                    Ender IO was build against API version rv2 and may or may not work with a newer version.
                 This may have caused the error. Try reproducing the crash WITHOUT this/these mod(s) before reporting it.
	Forestry : Info: The following plugins have been disabled in the config: magicalcrops
	AE2 Integration: IC2:ON, RotaryCraft:OFF, RC:ON, BuildCraftCore:ON, BuildCraftTransport:ON, BuildCraftBuilder:ON, RF:ON, RFItem:ON, MFR:OFF, DSU:ON, FZ:OFF, FMP:ON, RB:OFF, CLApi:OFF, Waila:ON, Mekanism:OFF, ImmibisMicroblocks:OFF, BetterStorage:OFF, OpenComputers:ON, PneumaticCraft:OFF
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 2 / 10; [EntityPlayerMP['WhatsTheBadNews'/264, l='world', x=439.82, y=50.00, z=-1923.70], EntityPlayerMP['Apollsyne'/909, l='world', x=424.52, y=50.00, z=-1920.87]]
	Is Modded: Definitely; Server brand changed to 'fml,forge'
	Type: Dedicated Server (map_server.txt)`

--- a/build.gradle
+++ b/build.gradle
@@ -1,4 +1,4 @@
-//version: 1646409286
+//version: 1650343995
 /*
 DO NOT CHANGE THIS FILE!
 
@@ -568,7 +568,7 @@ publishing {
                 artifact source: shadowJar, classifier: ""
             }
             if(!noPublishedSources) {
-                artifact source: sourcesJar, classifier: "src"
+                artifact source: sourcesJar, classifier: "sources"
             }
             artifact source: usesShadowedDependencies.toBoolean() ? shadowDevJar : devJar, classifier: "dev"
             if (apiPackage) {

--- a/src/main/java/toast/specialMobs/entity/EntitySpecialFishHook.java
+++ b/src/main/java/toast/specialMobs/entity/EntitySpecialFishHook.java
@@ -114,6 +114,7 @@ public class EntitySpecialFishHook extends Entity {
         if (!this.worldObj.isRemote) {
             if (this.angler == null || this.angler.isDead || this.getDistanceSqToEntity(this.angler) > 1024.0) {
                 this.setDead();
+                return;
             }
             Vec3 posVec = Vec3.createVectorHelper(this.posX, this.posY, this.posZ);
             Vec3 motionVec = Vec3.createVectorHelper(this.posX + this.motionX, this.posY + this.motionY, this.posZ + this.motionZ);
@@ -174,6 +175,7 @@ public class EntitySpecialFishHook extends Entity {
         this.rotationYaw = this.prevRotationYaw + (this.rotationYaw - this.prevRotationYaw) * 0.2F;
         if (this.isInWater()) {
             this.setDead();
+            return;
         }
         this.motionX *= 0.99;
         this.motionY *= 0.99;

--- a/src/main/java/toast/specialMobs/entity/EntitySpecialSpitball.java
+++ b/src/main/java/toast/specialMobs/entity/EntitySpecialSpitball.java
@@ -99,6 +99,7 @@ public class EntitySpecialSpitball extends Entity
         if (!this.worldObj.isRemote) {
             if (this.shootingEntity == null || this.shootingEntity.isDead || this.getDistanceSqToEntity(this.shootingEntity) > 1024.0) {
                 this.setDead();
+                return;
             }
             Vec3 posVec = Vec3.createVectorHelper(this.posX, this.posY, this.posZ);
             Vec3 motionVec = Vec3.createVectorHelper(this.posX + this.motionX, this.posY + this.motionY, this.posZ + this.motionZ);
@@ -153,6 +154,7 @@ public class EntitySpecialSpitball extends Entity
         this.rotationYaw = this.prevRotationYaw + (this.rotationYaw - this.prevRotationYaw) * 0.2F;
         if (this.isInWater()) {
             this.setDead();
+            return;
         }
         this.motionX *= 0.99;
         this.motionY *= 0.99;

--- a/src/main/java/toast/specialMobs/entity/ghast/Entity_SpecialGhast.java
+++ b/src/main/java/toast/specialMobs/entity/ghast/Entity_SpecialGhast.java
@@ -100,6 +100,7 @@ public class Entity_SpecialGhast extends EntityGhast implements ISpecialMob {
         /// Check for despawning.
         if (!this.worldObj.isRemote && this.worldObj.difficultySetting.getDifficultyId() == 0) {
             this.setDead();
+            return;
         }
         this.despawnEntity();
         this.prevAttackCounter = this.attackCounter;

